inertia/http.py

```diff
@@ -1,6 +1,7 @@
 from functools import wraps
 from http import HTTPStatus
 from json import dumps as json_encode
+from typing import Any, Callable
```

Comment:
Lazy annotations and `if TYPE_CHECKING` might be a nice way to avoid unessecary imports.